### ISSUE TITLE
Overleaf import-ability

### ISSUE THREAD

**Issue by darksonic37** [detiuaveiro/ua-thesis-template#2]

**Comment by fabiomaia** [comment]
I think we should reconsider this. Overleaf is increasingly more popular in the research community. Let's not be elitist and pretend everyone likes working at the CLI. It'd be a nice feature appreciated by most people.

Plus we're just talking about a single file, which seems to be the recommended way of configuring latexmk anyway. I dislike additional config files too, but there's a benefit in this case.

**Comment by LudeeD** [comment]
I suggest closing down this issue, since there is no real solution for this, besides sketchy hacks.

**Comment by LudeeD** [comment]
> Commit [1269078](https://github.com/fabiomaia/ua-thesis-template/commit/12690780b61e8238de628ba3498689ff3230903c) should add support for overleaf (please test).

It seems to be working

About the other file, my vote is to leave it out of this repo and update the documentation about this

**Comment by LudeeD** [comment]
Recently the NEEF - Núcleo de Estudantes de Engenharia Física <neef@aauav.pt> reached out to know more about this functionality and maybe do a workshop for them on how to do this.

How is the state on this project xD ?

**Comment by rj-jesus** [comment]
This project -i.e.- using the template with Overleaf? I just did a quick test and the instructions from https://github.com/detiuaveiro/ua-thesis-template#use-it-in-overleaf still seem to apply, with the caveat that the user needs to change the main document to `matter.tex` in Overleaf's settings, which we currently do not state.

It would perhaps be easier if we (or NEEF) created a template based on this repo on Overleaf so users could use/open it directly.

P.S. I'm under the impression [the issue with `ftmcount`](https://github.com/detiuaveiro/ua-thesis-template/issues/2#issuecomment-465199483) no longer applies.

**Comment by Jpfonseca** [comment]
Sorry to jump in the thread. I believe I opened #28  without checking this issue.

> I just did a quick test and the instructions from https://github.com/detiuaveiro/ua-thesis-template#use-it-in-overleaf still seem to apply, with the caveat that the user needs to change the main document to matter.tex in Overleaf's settings, which we currently do not state.

Yes, that's true. 

>It would perhaps be easier if we (or NEEF) created a template based on this repo on Overleaf so users could use/open it directly.

In April, after being contacted by NEEF too, I took time to upload the latest changes to the template to Overleaf so it can be easily used by colleagues from other departments. This has been documented in #28 too. In the future, all that is needed is to request the Overleaf team to update the template.

To promote a better track of updates across forks, I believe this repo should be transformed into a [template repo](https://github.com/detiuaveiro/ua-thesis-template/issues/28#issue-1193620334). It simplifies for people working on their documents to update their documents with the latest changes to the upstream project.
